hikari/__init__.py

```diff
@@ -39,23 +39,33 @@
 from hikari._about import __license__
 from hikari._about import __url__
 from hikari._about import __version__
+from hikari.applications import *
+from hikari.audit_logs import *
+from hikari.channels import *
 from hikari.config import *
+from hikari.embeds import *
+from hikari.emojis import *
 from hikari.errors import *
 from hikari.events import *
+from hikari.files import Bytes
+from hikari.files import File
+from hikari.files import LazyByteIteratorish
+from hikari.files import Pathish
+from hikari.files import Rawish
+from hikari.files import Resourceish
+from hikari.files import URL
 from hikari.impl.bot import BotApp as Bot
 from hikari.impl.rest import RESTApp as REST
-from hikari.models import *
-from hikari.utilities.files import File
-from hikari.utilities.files import LazyByteIteratorish
-from hikari.utilities.files import Pathish
-from hikari.utilities.files import Rawish
-from hikari.utilities.files import Resourceish
-from hikari.utilities.snowflake import SearchableSnowflakeish
-from hikari.utilities.snowflake import SearchableSnowflakeishOr
-from hikari.utilities.snowflake import Snowflake
-from hikari.utilities.snowflake import Snowflakeish
-from hikari.utilities.snowflake import SnowflakeishOr
-from hikari.utilities.snowflake import Unique
-from hikari.utilities.undefined import UNDEFINED
-from hikari.utilities.undefined import UndefinedNoneOr
-from hikari.utilities.undefined import UndefinedOr
+from hikari.intents import *
+from hikari.invites import *
+from hikari.iterators import *
+from hikari.messages import *
+from hikari.permissions import *
+from hikari.presences import *
+from hikari.sessions import *
+from hikari.snowflakes import *
+from hikari.traits import *
+from hikari.undefined import *
```

Comment:
This imports the `count` function to the root project. Should be better have it how it was before